iphone/cli/commands/_buildModule.js

```diff
@@ -237,6 +252,60 @@ iOSModuleBuilder.prototype.processLicense = function processLicense() {
 };
 
 iOSModuleBuilder.prototype.processTiXcconfig = function processTiXcconfig(next) {
+	const srcFile = path.join(this.projectDir, this.moduleName + '.xcodeproj', 'project.pbxproj');
+	const xcodeProject = xcode.project(path.join(this.projectDir, this.moduleName + '.xcodeproj', 'project.pbxproj'));
+	const contents = fs.readFileSync(srcFile).toString();
```

Comment:
FYI, when reading a file, you can avoid converting from `Buffer` to `String` (calling `toString()`), by passing in an encoding argument to the readFile call: `fs.readFileSync(srcFile, 'utf8');`